shared/modules/persistence_module.go

```diff
@@ -72,7 +66,7 @@ type PersistenceWriteContext interface {
 	SubtractPoolAmount(name string, amount string) error
 	SetPoolAmount(name string, amount string) error
 
-	InsertPool(name string, address []byte, amount string) error // TODO (Andrew) remove address from pool #149
+	InsertPool(name string, address []byte, amount string) error
```

Comment:
Per the TODO removed, do we still want to keep passing in an address for the pool?

Reply:
No we don't, but this already has a `TODO` for the persistence module which is out of scope for this issue:

`// TODO(andrew): remove address param`